COPYING

```diff
@@ -1,54 +1,5 @@
-tinycompress is provided under a dual BSD/LGPLv2.1 license.  When using or
-  redistributing tinycompress, you may do so under either license.
+tinycompress is provided under:
+	SPDX-License-Identifier: LGPL-2.1-only OR BSD-3-Clause
```

Comment:
why not use the same notation as in the rest of the files

SPDX-License-Identifier: (LGPL-2.1-only OR BSD-3-Clause)

Reply:
this should, thanks for pointing